server/www/templates/modal/modal.html

```diff
@@ -1,14 +1,18 @@
 <div ng-class="{ 'modal-visible': classVisible, 'modal-detached': classDetached }">
-    <div class="modal">
-        <div class="modal-window" ng-style="{ top : modalOffset }">
-            <a ng-click="closeModal('standard')" ng-if="showCloseButton" class="close modal-trigger"></a>
-
-            <h3 ng-if="title">{{ title | translate }}</h3>
-            <div ng-if="title" class="divider padded"></div>
+    <div class="modal" style="display:block">
```

Comment:
Any way to avoid the inline style here? I think this `.modal` div should be combined with the div above. If the modal-visible / modal-detached classes are no longer relevant maybe use ng-show/ng-hide instead?